### ISSUE TITLE
Which is Plagiarism: Fashion Image Retrieval based on Regional Representation for Design Protection

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#95]
# TL;DR
A model for judging plagiarism is proposed. Since plagiarism often involves altering a part of a garment, the feature map is divided into regions, and in each using triplet loss. Plagiarism whose design is the same but the color is different  be detected.
![image](https://user-images.githubusercontent.com/24524018/92561663-9454b780-f2af-11ea-9ebd-2472ec6a1e6f.png)

# Why it matters:

# Paper URL
https://openaccess.thecvf.com/content_CVPR_2020/papers/Lang_Which_Is_Plagiarism_Fashion_Image_Retrieval_Based_on_Regional_Representation_CVPR_2020_paper.pdf 

# Submission Dates(yyyy/mm/dd)

# Authors and institutions

# Methods

# Results

# Comments
